khelo/assets/locales/app_en.arb

```diff
@@ -105,8 +105,25 @@
   "@_HOME": {
   },
   "home_screen_title": "Home",
-  "home_screen_no_matches_title": "No matches are running currently",
-  "home_screen_no_matches_description_text": "Running matches will be shown here, Stay tuned for some intense matches.",
+  "home_screen_live_title": "Live",
+  "home_screen_upcoming_title": "Upcoming",
+  "home_screen_winning_title": "Winning",
```

Comment:
winning title doen't make any sense for me, can we replace it with "Finished" or "Completed"?